{|
  | '''Warning'''
  * This is a '''readonly''' and '''text-based''' archive of a deprecated wiki.
  * '''Images''' and '''attachments''' have been removed to conserve space.
  * '''Links''' may not work and there may be formatting issues.
  * A '''compressed''' version with images and the original syntax is in the repo '''Releases'''.
|}

__TOC__

== Graphical identification of cancer-associated  gene sub-networks based on small proteomics data sets ==
=== Author ===
{| class="wikitable"
|-
| <nowiki>[[http://bioinformatics.tn/|Karim Mezhoud]]</nowiki>
|}
UR04CNSTN01 - Bio-computing Unit, Life Science Department, National Center for Nuclear Sciences and Technologies (<nowiki>[[http://www.cnstn.rnrt.tn/| CNSTN]]</nowiki>). <nowiki>[[http://www.biotechpole.rnu.tn/en/default.aspx|Sidi Thabet BioTechPole]]</nowiki>, 2020, Ariana, Tunisia.
Email: kmezhoud(at)bioinformatics.tn

=== Description ===
These supplemental files allow reader to follow the Methodology described in the paper accepted in  <nowiki>[[http://www.liebertpub.com/OMI|OMICS: A Journal of integrative Biology]]</nowiki>.

=== Figures ===
{| class="wikitable"
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Figure+1.eps|Figure 1.eps]]</nowiki>: Legend is available in main text.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Figure+2.eps|Figure 2.eps]]</nowiki>: Legend is available in main text.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Figure+3.eps|Figure 3.eps]]</nowiki>: Legend is available in main text.
|}

=== Supplemental files ===
{| class="wikitable"
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Supplemental_Tutorial.pdf|Supplemental_Tutorial.pdf]]</nowiki>: This tutorial describes the main steps used to construct an enriched network from a list of genes with their expression values.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=only_modulated_symbol.txt|only_modulated_symbol.txt]]</nowiki>: The list of select genes after proteomics analysis.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Correlation_Network.txt|Correlation_Network.txt]]</nowiki>: This file connect genes (pp) that have expressions correlation.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Fold_Change.txt|Fold_change.txt]]</nowiki>: This file attributes genes (nodes) with their fold change.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=FDR.txt|FDR.txt]]</nowiki>: This file attributes genes (nodes) with their False Discovery Rate (FDR).
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Correlation%2BCancers.txt|Correlation+Cancers.txt]]</nowiki>: This file attributes values to interactions (Edges). There is two types of interactions: gene-gene and gene-cancer.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Cancer_Network.txt|Cancer_Network.txt]]</nowiki>: This file connects genes with cancers which are modulated.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Cancer_frequency.txt|Cancer_frequency.txt]]</nowiki>: This file attributes Cancers (nodes) with their frequency.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Edge_Disease.txt|Edge_Disease.txt]]</nowiki>: This file attributes Cancers (nodes) with their frequency.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=network.cys|network.cys]]</nowiki>: This file is the Cytoscape session of this work. It is readable only by Cytoscape software.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=proteinEXP.txt|proteinEXP.txt]]</nowiki>: This is an example of proteomic data that used for this work. It is consist of 3 replicates and two conditions.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Patient-profile.png|Patient-profile.png]]</nowiki>: The general characteristics of the patients/volunteers are described here.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=only_modulatedEXP.txt|only_modulatedEXP.txt]]</nowiki>: This file was submitted to the <nowiki>[[http://www.cbioportal.org/public-portal/|cBio Cancer genomics portal]]</nowiki> (Figure 2, supplemental Tutorial).
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Supplemental_Table1.pdf|Supplemantal_Table1.pdf]]</nowiki>:  The features of the discussed bioinfomatic tools.
|-
| <nowiki>[[https://wiki.ubuntu.com/kmezhoud/supplemental_material?action=AttachFile&do=get&target=Supplemental_Table+2.pdf|Supplemental_Table2.pdf]]</nowiki>: Non exhaustive list of important protein from Figure 2.
|-
| <nowiki>[[http://youtu.be/7E4Q_JyTyWw|Supplemental_Video]]</nowiki>: This video shows all steps with comments.
|}

----
CategoryScience
